app/views/diversities/_claim_type.html.slim

```diff
@@ -1,7 +1,7 @@
 fieldset
   = f.input :claim_type,
     as: :radio_buttons,
-    include_hidden: false,
+    readonly: nil,
```

Comment:
Have you intentionally overwritten `include_hidden: false` on this one?

Reply:
it was a duplicate